### DISCUSSION TITLE
chromium not starting

### DISCUSSION THREAD

**Discussion by jjuzwik** [debloper/piosk#54] [Q&A]
Hi, thanks very much for making PIOSK! I have been using it quite happily for a couple weeks. But today, I decided to wipe, and re-install the pi image (and kiosk), and now, PIOSK is not opening the browser on boot. The web interface is working, and able to reboot etc, but it's just not launching chromium. I am able to manually open chromium from the desktop just fine.

This is a pi 5, running the latest RPI OS 32-bit. Same hardware that was running flawlessly for many weeks.

Do you have any ideas?

**Comment by debloper** [comment]
My guess is that it's related to the `labwc` change.

More details: #52 & #53 

Please confirm if that's the case.

**Comment by jjuzwik** [comment]
I read both of those threads and wasn't able to see a way to verify if it
is related or not.

How do i check?

Thx,
Josh

**Reply by creative0100** [reply]
whats selected in raspi-config? My new installation wouldnt open until I changed it to Wayland there.
